lib/Models/Terria.js

```diff
@@ -86,12 +85,6 @@ var Terria = function(options) {
     throw new DeveloperError("options.baseUrl is required.");
   }
 
-  options.languageOverrides = defined(options.languageOverrides)
-    ? options.languageOverrides
-    : {};
-
-  this.language = combine(options.languageOverrides, language);
```

Comment:
As mentioned in https://github.com/TerriaJS/terriajs/issues/3785#issuecomment-557305148 we should still leave this in, as the overrides are still used for function-specific-overrides

Reply:
can you please check if this part is OK now, after last commit

Reply:
Thanks @zoran995 ! That's exactly what we need.